test/tests/redfish10/test_redfish10_api_updateservice.py

```diff
@@ -0,0 +1,32 @@
+'''
+Copyright 2017 Dell Inc. or its subsidiaries.  All Rights Reserved.
+'''
+
+import fit_path  # NOQA: unused import
+import fit_common
+
+NODECATALOG = fit_common.node_select()
+
+
+# Select test group here using @attr
+from nose.plugins.attrib import attr
+
+
+@attr(all=True, regression=True, smoke=True)
+class redfish10_api_updateservice(fit_common.unittest.TestCase):
+    def test_redfish_v1_updateservice_actions_updateservicesimpleupdate_post(self):
+        nodeid = NODECATALOG[0]
+        fit_common.cancel_active_workflows(nodeid)
+        on_payload = {"ImageURI": "/dummy.exe", "Targets": [nodeid]}
+        api_data = fit_common.rackhdapi('/redfish/v1/UpdateService/Actions/UpdateService.SimpleUpdate',
+                                        action='post', payload=on_payload)
+        self.assertEqual(api_data['status'], 201,
+                         'Incorrect HTTP return code, expected 201, got:' + str(api_data['status']))
+        task_data = fit_common.rackhdapi(api_data['json']['@odata.id'])
+        self.assertEqual(task_data['status'], 200, "No task ID found ")
+        self.assertIn(task_data['json']['TaskState'], ["Running", "Pending", "Completed", "Exception"],
+                      "Bad task state for node:" + nodeid)
```

Comment:
Can you print out the bad task state received in the error message?  Helps in debugging.
"Bad task state {} for node {}".format(task_data[....}, nodeid)